packages/lit-dev-content/samples/examples/motion-simple/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "/samples/base.json",
+  "title": "simple",
```

Comment:
Here and throughout -- WDYT about slightly more descriptive names? Like simple -> "Sliding circle", grid -> "Grid layout transitions", etc.?